docs/api/pycalphad.rst

```diff
@@ -6,23 +6,33 @@ Subpackages
 
 .. toctree::
 
+    pycalphad.codegen
     pycalphad.core
     pycalphad.io
     pycalphad.plot
+    pycalphad.tests
 
 Submodules
 ----------
 
-pycalphad\.model module
------------------------
+pycalphad.model module
+----------------------
 
 .. automodule:: pycalphad.model
     :members:
     :undoc-members:
     :show-inheritance:
 
-pycalphad\.variables module
----------------------------
+pycalphad.refdata module
```

Comment:
We should probably remove this documentation and module. It's not a blocker, but either remove or open an issue to put it in the deprecation cycle so we don't forget.

Reply:
Defer to 0.8